virttest/qemu_vm.py

```diff
@@ -948,9 +948,9 @@ def add_memorys(devices, params):
                 min_mem_size_m = params.get("vm_mem_minimum")
                 min_mem_size_m = float(normalize_data_size(min_mem_size_m))
                 if mem_size_m < min_mem_size_m:
-                    logging.info("Guest min memory has to be %s"
-                                 % min_mem_size_m)
-                    mem_size_m = min_mem_size_m
+                    raise exceptions.TestCancel("Guest min memory has to be %s"
```

Comment:
try:
        self.devices, self.spice_options = self.make_create_command()
         ....      
    except Exception:
        for nic in self.virtnet:
            self._nic_tap_remove_helper(nic)
            utils_misc.log_last_traceback('Fail to create qemu command:')
            raise virt_vm.VMStartError(self.name, 'Error occurred while '
                                                       'executing make_create_command(). '
                                                       'Check the log for traceback.')

TestCancel will be handled here( in line 2725), and raise VMStartError finally; suggest not to handle TestCancel here, raise it to make case fail reason more intuitive.

Reply:
updated, thanks.